assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -92,6 +93,7 @@ che.workspace.maven_server_java_options=-XX:MaxRAM=128m -XX:MaxRAMFraction=1 -XX
 che.workspace.terminal_linux_amd64=${che.home}/lib/linux_amd64/terminal
 che.workspace.terminal_linux_arm7=${che.home}/lib/linux_arm7/terminal
 
+# The location of the exec
```

Comment:
Maybe it's better to remove such unused property instead of adding documentation for it? )
`che.workspace.terminal_linux_amd64` and `che.workspace.wsagent_java_options` seems outdated as well.

Reply:
ok, good to know